pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -226,6 +226,10 @@ public CmdTopics(Supplier<PulsarAdmin> admin) {
 
         jcommander.addCommand("set-replicated-subscription-status", new SetReplicatedSubscriptionStatus());
 
+        jcommander.addCommand("get-replication-clusters", new GetReplicationClusters());
```

Comment:
Please also add unit test in `PulsarAdminToolTest`

Reply:
Ok, done